diff -ru a/CPP/7zip/UI/Common/ArchiveExtractCallback.cpp b/CPP/7zip/UI/Common/ArchiveExtractCallback.cpp
--- a/CPP/7zip/UI/Common/ArchiveExtractCallback.cpp	2025-06-29 11:00:00.000000000 +0200
+++ b/CPP/7zip/UI/Common/ArchiveExtractCallback.cpp	2025-08-02 21:43:07.673526335 +0200
@@ -2012,10 +2012,16 @@
     path is relative to root folder of extracted items
     if (!link.isRelative), then (path == link.LinkPath)
     */
-    if (!IsSafePath(path, link.Is_WSL()))
+    if (!IsSafePath(path, link.Is_WSL())) {
+      // install dmgs have often /Application links; don't abort but don't create link either
+      if (_ntOptions.SymLinks_AllowDangerous.Val) {
+        return S_OK;
+      }
+
       return SendMessageError2(0, // errorCode
           "Dangerous link path was ignored",
           us2fs(_item.Path), us2fs(link.LinkPath));
+    }
   }
 
   FString target; // target path that will be stored to link field
