diff -ru a/CPP/7zip/UI/Common/ArchiveExtractCallback.cpp b/CPP/7zip/UI/Common/ArchiveExtractCallback.cpp
--- a/CPP/7zip/UI/Common/ArchiveExtractCallback.cpp	2025-08-02 20:42:00.000000000 +0200
+++ b/CPP/7zip/UI/Common/ArchiveExtractCallback.cpp	2025-08-30 21:16:43.227294663 +0200
@@ -2199,8 +2199,6 @@
       if (!link.isRelative), then (path == link.LinkPath)
       */
       isDang = false;
-      if (!IsSafePath(path, link.Is_WSL()))
-        isDang = true;
     }
     const char *message = NULL;
     if (isDang)
