diff -ru a/CPP/Windows/System.h b/CPP/Windows/System.h
--- a/CPP/Windows/System.h	2023-03-22 20:00:00.000000000 +0100
+++ b/CPP/Windows/System.h	2023-08-07 18:54:51.010000194 +0200
@@ -105,7 +105,7 @@
   }
   
   void CpuZero() { }
-  void CpuSet(unsigned cpuIndex) { UNUSED_VAR(cpuIndex); }
+  void CpuSet(unsigned cpuIndex) { UNUSED_VAR(cpuIndex) }
   int IsCpuSet(unsigned cpuIndex) const { return (cpuIndex < numSysThreads) ? 1 : 0; }
 
   BOOL SetProcAffinity() const
